misc-tools/import-contest.in

```diff
@@ -130,10 +130,11 @@ import_file('groups', ['groups.json', 'groups.tsv'])
 if import_file('organizations', ['organizations.json']):
     # Also import logos if we have any
     # We prefer the 64x64 logo. If it doesn't exist, accept a generic logo (which might be a SVG)
-    import_images('organizations', 'logo', ['^logo\.64\..*$', '^logo\.[a-z]*$'])
+    # We also prefer PNG/SVG before JPG
+    import_images('organizations', 'logo', ['^logo\.64x\d+\.png$', '^logo\.(png|svg)$', '^logo\.64x\d+\.jpg$', '^logo\.jpg$'])
 if import_file('teams', ['teams.json', 'teams2.tsv']):
-    # Also import photos of we have any
-    import_images('teams', 'photo', ['^photo\.[a-z]*$'])
+    # Also import photos if we have any, but prefer PNG and PNG over JPG
+    import_images('teams', 'photo', ['^photo\.(png|svg)$', '^photo\.jpg$'])
```

Comment:
Why prefer PNG over JPG? If these are indeed photos, then I think that JPG should normally offer better quality/size ratio.

Reply:
Fixed.